internal/agents/grok/signin.go

```
@@ -0,0 +1,52 @@
+// credential so the caller can report who signed in.
+func (s Store) SignIn(ctx context.Context, client *http.Client, out io.Writer) (CredentialInfo, error) {
+	config, err := discoverOAuthConfig(ctx, client)
+	if err != nil {
+		return CredentialInfo{}, err
+	}
+	code, err := oauthdevice.RequestCode(ctx, client, config, time.Now())
+	if err != nil {
+		return CredentialInfo{}, err
+	}
+	if code.VerificationURIComplete != "" {
+		fmt.Fprintf(out, "Open %s to sign in to your Grok subscription", code.VerificationURIComplete)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)